calculator/main.py

```diff
@@ -37,14 +61,21 @@ def add_new_operation(calc, operation):
     :param operation: Dict with the single operation to be added.
                       ie: {'add': add_function}
     """
-    pass
+    
+    #calc['operations'] = dict(list(calc['operations'].items()) + list(operation.items()))
+    
+    newname = list(operation.keys())[0]
```

Comment:
Nice...thanks! I didn't realize that the test was passing an invalid param until your comment...spent forever trying to fix that.